src/robot/libdocpkg/model.py

```diff
@@ -12,13 +12,28 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
+from inspect import isclass
 from itertools import chain
 import json
 import re
+try:
+    from enum import Enum
+except ImportError:  # Standard in Py 3.4+ but can be separately installed
+    class Enum(object):
+        pass
+
+try:
+    from typing_extensions import TypedDict
+
+    class _TypedDictDummy(TypedDict):
+        pass
+    TypedDictType = type(_TypedDictDummy)
+except ImportError:
+    TypedDictType = None
```

Comment:
conditional import of typing or typing_extensions

Reply:
from typing import TypedDict
type(TypedDict('X', {}))
<class 'typing._TypedDictMeta'>

Reply:
import typing_extensions with old python versions